cli/compose/loader/loader_v1_test.go

```diff
@@ -0,0 +1,37 @@
+package loader
+
+import (
+	"testing"
+	"github.com/stretchr/testify/require"
+)
+
+
+func TestUnsupportedVersion(t *testing.T) {
+	_, err := loadYAML(`
+version: "0.1"
+services:
+  foo:
+    image: busybox
+`)
+	require.Error(t, err)
+	require.Contains(t, err.Error(), "version")
+
+	_, err = loadYAML(`
```

Comment:
Why testing the same error again?

Reply:
you are right, I'll remove :stuck_out_tongue: